README.md

```diff
@@ -36,6 +36,9 @@ By default, the environment will be **development**, but you can easily change i
 #### Debugging
 As we know, a NodeJS application is not something easy to debug and because of that we've added the `--inspect` flag to make it simpler. Chrome DevTools will get started when running your app using the start script (`npm start`), making your debugging easier.
 
+#### REPL console
+We can use a node console with `npm run console`. There your service objects are exposed as _servicename_ + "Service". Let's suppose that we have a service `users` which has a function `getAll`. In your console you can call `usersService.getAll()` and see the result. Note that this work also with functions that return promises! To exit the console, use `.exit`.
+
```

Comment:
There's a typo here: it should be `Note that this works also ...`. And I'd also delete the comma in the last sentence: `To exit the console use ` `.exit`.

Reply:
thank you!